nss-cohort-36/bangazon-api-virtual_magicians
# Jl last api changes

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-last-api-changes`
4. pull down, check out, and go to react pull request
5. when merging notice the datetimefield() in models/order.py - it must have (auto_now=False, auto_now_add=True) in the parentheses to function correctly

# Checklist:
